app/controllers/devices_controller.rb

```diff
@@ -1,48 +1,50 @@
 class DevicesController < ApplicationController
```

Comment:
@dinakartejat like employees_controller, you should have only search action method in devices_controller. should not modify existing action methods like index and all